<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
    <item>
     <widget class="QTextEdit" name="map">
      <property name="font">
       <font>
        <family>Inconsolata</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTextEdit" name="log">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="buttonStartAi">
          <property name="text">
           <string>Start AI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonStopAi">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop AI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRestart">
          <property name="text">
           <string>Regenerate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkboxEmoji">
          <property name="text">
           <string>Emoji</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
